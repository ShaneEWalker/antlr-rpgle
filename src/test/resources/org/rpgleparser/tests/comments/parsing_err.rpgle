   .        // Print a Report 
33723           EMAILR ( Parm_Lda : ExcelYN : PASSWD :                        // CALL PGM
33723           EMAIL : INDEX : PGMH );
   .           // Job submit msg