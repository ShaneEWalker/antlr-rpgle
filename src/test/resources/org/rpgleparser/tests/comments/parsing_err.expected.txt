/*===TOKENS===*/
HIDDEN:LEAD_WS5_Comments <   . >
HIDDEN:COMMENTS          <       //>
HIDDEN:COMMENTS_TEXT     <Print a Report>
HIDDEN:LEAD_WS5_Comments <33723>
ID                       <EMAILR>
'('                      <(>
ID                       <Parm_Lda>
COLON                    <:>
ID                       <ExcelYN>
COLON                    <:>
ID                       <PASSWD>
COLON                    <:>
HIDDEN:FREE_COMMENTS     <//>
HIDDEN:COMMENTS_TEXT_HIDDEN< CALL PGM          >
HIDDEN:COMMENTS_EOL_HIDDEN<
33723 >
ID                       <EMAIL>
COLON                    <:>
ID                       <INDEX>
COLON                    <:>
ID                       <PGMH>
')'                      <)>
FREE_SEMI                <;>
HIDDEN:LEAD_WS5_Comments <   . >
HIDDEN:COMMENTS          <          //>
HIDDEN:COMMENTS_TEXT     <Job submit msg>
/*===TREE===*/
(r
  (statement
    (free
      (baseExpression
        (op
          (op_callp
            (identifier (free_identifier (idOrKeyword EMAILR)))
            (
            (expression (identifier (free_identifier (idOrKeyword Parm_Lda))))
            :
            (expression (identifier (free_identifier (idOrKeyword ExcelYN))))
            :
            (expression (identifier (free_identifier (idOrKeyword PASSWD))))
            :
            (expression (identifier (free_identifier (idOrKeyword EMAIL))))
            :
            (expression (identifier (free_identifier (idOrKeyword INDEX))))
            :
            (expression (identifier (free_identifier (idOrKeyword PGMH))))
            )
          )
        )
      )
      ;
    )
  )
)
/*======*/